Containerfile

```diff
@@ -0,0 +1,34 @@
+FROM --platform=linux/amd64 debian:bookworm-slim as devtools
+
+RUN apt-get update && apt-get install -y --no-install-recommends clang libclang-rt-dev
+RUN apt-get update && apt-get install -y --no-install-recommends cmake ninja-build
+RUN apt-get update && apt-get install -y --no-install-recommends libgrpc++-dev libprotobuf-dev protobuf-compiler-grpc
+RUN apt-get update && apt-get install -y --no-install-recommends libpq-dev postgresql-client
```

Comment:
Probably will speed up a bit of the build by combining this into one (or at least just having one `apt-get update`).